x-pack/packages/kbn-elastic-assistant/impl/assistant/index.test.tsx

```diff
@@ -142,7 +142,8 @@ describe('Assistant', () => {
   });
 
   describe('persistent storage', () => {
-    it('should refetchCurrentUserConversations after settings save button click', async () => {
+    // Settings Modal is not available when `v2KnowledgeBaseEnabled` is true
```

Comment:
Since you will revert the feature flag back to false, I will fix this in [my PR lifting the feature flag](https://github.com/elastic/kibana/pull/198178). Please unskip when you revert flag to false.